core/server/data/migrations/versions/4.9/01-add-default-product-portal-products.js

```diff
@@ -0,0 +1,49 @@
+const {createTransactionalMigration} = require('../../utils');
```

Comment:
Is there a reason this is 4.9 instead of 4.8?

Reply:
None other than me totally missing 4.8 is the next release but just has new migrations 🤦 Updating

Reply:
Updated, thanks for catching 😄